### PR TITLE
Updated embedded Mailchimp form to reflect new list ID

### PR DESCRIPTION
**Pull request by vossisboss**
An attempt to consolidate Mailchimp lists for our Wagtail Newsletters went awry and I needed to recreate the list. I've updated the embedded form in the newsletter index page template to include the new audience ids.

I realized a bit belatedly that I could have just switched out the audience ids rather that replacing the whole form. So the code formatting is a bit different now. I'll correct anything that fails the style check, but if there is anything else that needs to be fixed, please let me know.

### CODE REVIEW

File: wagtailio/newsletter/templates/newsletter/newsletter_index_page.html
Comment (Stormheg) [inline wagtailio/newsletter/templates/newsletter/newsletter_index_page.html:19]: Could you indent this all one tab so it matches the original indentation? It makes the diff less noisy as a lot of html didn't  change.

File: wagtailio/newsletter/templates/newsletter/newsletter_index_page.html
Comment (Stormheg) [inline wagtailio/newsletter/templates/newsletter/newsletter_index_page.html:20]: Different css file, but the file contents appear to be really similar. Looks safe.

**Review by Stormheg — COMMENTED** [review]
Looks fine @vossisboss, but could you fix the indentation before we merge this please?

File: wagtailio/newsletter/templates/newsletter/newsletter_index_page.html
Comment (vossisboss) [reply wagtailio/newsletter/templates/newsletter/newsletter_index_page.html:20]: They probably have an updated version since the last time we generated this form. Thank you for taking a close look at it!

File: wagtailio/newsletter/templates/newsletter/newsletter_index_page.html
Comment (vossisboss) [reply wagtailio/newsletter/templates/newsletter/newsletter_index_page.html:19]: Will do.

**Comment by vossisboss** [comment]
@Stormheg I cleaned up the indentation to more closely match the original. There were a few spots that I thought could be a little cleaner though with some minor changes. Please let me know what you think.

**Review by Stormheg — APPROVED** [review]
Thanks for cleaning that up! 🚀

**Comment by Stormheg** [comment]
Approved, leaving it up to you to merge. Does this need to be applied to the [`staging`](https://github.com/wagtail/wagtail.org/tree/staging) branch too?

**Comment by vossisboss** [comment]
@Stormheg I think it should go to the staging branch too. Alas, I don't have merge permissions for this repository so I'll have to poke around and find out who does.

**Comment by Stormheg** [comment]
Oh I presumed you had the rights to merge this. I can do it for you and cherry-pick this into staging. I don't have the rights to deploy the change as I don't work for Torchbox anymore. Probably best to ask Karl.

**Comment by Stormheg** [comment]
Cherry picked on staging in https://github.com/wagtail/wagtail.org/commit/32e532292630c08c9660a195a872345de74311fc

**Comment by vossisboss** [comment]
Thanks @Stormheg ! I don't have deploy powers either. So @kaedroho or @RealOrangeOne, could you help us out with getting this merge live when you have a moment?

**Comment by kaedroho** [comment]
Just to check, this new audience ID will add uses to "This Week In Wagtail", not the "Wagtail Newsletter" list? (which is separate and currently used in the footer)

**Comment by vossisboss** [comment]
Yes. It will add subscribers to This Week in Wagtail. The link worked for me in my local version and I added one of my test emails to the list, so you're welcome to test it in staging if you'd like. I can confirm whatever email is added to the list if you don't have MailChimp access.

**Pull request merged**
